src/WpfMath/Controls/FormulaControl.xaml.cs

```diff
@@ -111,6 +119,20 @@ public FormulaControl()
         {
             InitializeComponent();
         }
+        
+        protected override void OnRender(DrawingContext drawingContext)
+        {
+            base.OnRender(drawingContext);
+            
+            if(IsCharacterSmoothingEnabled)
+            {
+                base.VisualTextRenderingMode = TextRenderingMode.ClearType;
+                base.VisualTextHintingMode = TextHintingMode.Fixed;
+                base.VisualEdgeMode = EdgeMode.Aliased;
+                base.VisualClearTypeHint = ClearTypeHint.Enabled;
+            }
+       
+        }
```

Comment:
I can't see the purpose of setting these properties *after* calling `base.OnRender`.

If they affect render, then they should be set before calling `base.OnRender`. If they don't, then they should be set in the dependency property callback instead.